### PR TITLE
Bug fix for update circuit billing type

### PR DESCRIPTION
**Pull request by dihan0604**

### CODE REVIEW

**Comment by hovsepm** [comment]
@dihan0604 Any test that verifies new functionality?

**Comment by markcowl** [comment]
@dihan0604 Please add tests, or we will not be able to merge this

**Comment by dihan0604** [comment]
I have trouble to record the tests. I have sent email to azsdkcode for help.

**Comment by dihan0604** [comment]
Added tests

File: src/ServiceManagement/Services/Commands.Test/ExpressRoute/AzureDedicatedCircuitTests.cs [hunk 1]
```diff
@@ -91,8 +93,27 @@ public void NewAzureDedicatedCircuitSuccessful()
...
+
+            var tUpdate = new Task<ExpressRouteOperationStatusResponse>(() => expectedUpdateStatus);
+            tUpdate.Start();
+
+            //Task<ExpressRouteOperationStatusResponse> NewAsync(DedicatedCircuitNewParameters parameters, CancellationToken cancellationToken);
+            dcMock.Setup(f => f.NewAsync(It.Is<DedicatedCircuitNewParameters>(x => x.Bandwidth == bandwidth && x.CircuitName == circuitName && x.Location == location && x.ServiceProviderName == serviceProviderName), 
+                It.IsAny<CancellationToken>())).Returns((DedicatedCircuitNewParameters param, CancellationToken cancellation) => tNew);
             dcMock.Setup(f => f.GetAsync(It.Is<string>(sKey => sKey == serviceKey), It.IsAny<CancellationToken>())).Returns((string sKey, CancellationToken cancellation) => tGet);
+            
+            //Task<ExpressRouteOperationStatusResponse> UpdateAsync(string serviceKey, DedicatedCircuitUpdateParameters parameters, CancellationToken cancellationToken);
```
Comment (hovsepm) [inline src/ServiceManagement/Services/Commands.Test/ExpressRoute/AzureDedicatedCircuitTests.cs]: @dihan0604 Please remove commented code.

File: src/ServiceManagement/Services/Commands.Test/ExpressRoute/AzureDedicatedCircuitTests.cs [hunk 2]
```diff
@@ -91,8 +93,27 @@ public void NewAzureDedicatedCircuitSuccessful()
...
+            ExpressRouteOperationStatusResponse expectedUpdateStatus = new ExpressRouteOperationStatusResponse()
+            {
+                HttpStatusCode = HttpStatusCode.OK,
+                Data = serviceKey
+            };
+
+            var tUpdate = new Task<ExpressRouteOperationStatusResponse>(() => expectedUpdateStatus);
+            tUpdate.Start();
+
+            //Task<ExpressRouteOperationStatusResponse> NewAsync(DedicatedCircuitNewParameters parameters, CancellationToken cancellationToken);
```
Comment (hovsepm) [inline src/ServiceManagement/Services/Commands.Test/ExpressRoute/AzureDedicatedCircuitTests.cs]: @dihan0604 Please remove commented code.

**Comment by markcowl** [comment]
on-demand run here:  http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/693/

**Pull request merged**